### PR TITLE
update max-backups desc

### PR DESCRIPTION
**Pull request by glkappe**
it's int type.
 [FATAL] [terror.go:276] ["unexpected error"] [error="toml: cannot load TOML value of type string into a Go integer"]

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
@glkappe Would you please resolve the conflicts of this PR? Thanks

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -251,7 +251,8 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 日志文件的大小限制。
 + 默认值：300
 + 单位：MB
-+ 最大设置上限为 4GB。
++ 最大设置上限为 4096。
+
```
Comment (TomShawn) [inline tidb-configuration-file.md:255]:
```suggestion
```

**Review by qiancai — APPROVED** [review]

**Review by lzmhhh123 — APPROVED** [review]

**Comment by qiancai** [comment]
@glkappe We are not maintaining docs for release 2.1, 3.0, and 3.1 now, so please note that this PR will be only cherry-picked for 4.0 and later releases.

**Pull request merged**